### PR TITLE
Update SAP OPA setup

### PR DESCRIPTION
**Pull request by zweedao**

### CODE REVIEW

**Comment by zweedao** [comment]
quick fix @graceliching

File: connectors/sap.md [hunk 1]
```diff
@@ -100,9 +100,9 @@ The below configuration in the distribution model will enable SAP to send DEBMAS
 
 ### Configure Workato On-prem Agent (OPA) on SAP server
 #### Step 6: Download Workato On-prem Agent (OPA)
-For IDoc to work with Workato, you need to install a Workato On-prem Agent (OPA) on your SAP server.
+For SAP connector to work with Workato, you need to install a Workato On-prem Agent (OPA) on your SAP server, or on a separate Virtual Machine that can reach your SAP server by being in the same domain and not outside of the firewall.
```
Comment (graceliching) [inline connectors/sap.md]: Small change: 
For the SAP connector to work with Workato, you need to install a Workato On-prem Agent (OPA) on your SAP server, or on a separate Virtual Machine that can reach your SAP server in the same domain and not outside of the firewall.

**Comment by zweedao** [comment]
@graceliching ok I paraphrased a bit

**Comment by zweedao** [comment]
can you help merge @graceliching

**Pull request merged**